kong/kong.lua

```diff
@@ -1,4 +1,4 @@
--- Kong, the biggest ape in town
+-- Kong, the biggest ape in town 1
```

Comment:
But from there you cannot chose which branch to trigger the build from, which is useless.